client/js/components/StageTwo/StageTwo.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from "react";
+import {
+  ScrollView,
+  View,
+  Text,
+  Image,
+  TouchableOpacity,
+  Dimensions
+} from "react-native";
+import * as Progress from "react-native-progress";
+import styles from "./styles";
+import theme from "../../config/styles";
+export default class StageTwo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answer: null
+    };
+  }
+  answerTrue() {
+    this.setState({
+      answer: true
+    });
+  }
+  answerFalse() {
+    this.setState({
+      answer: false
+    });
+  }
```

Comment:
Change your both answer function to 
`
toggleAnswer = () => {this.setState({ answer: !this.state.answer })}
`